tests/functional/test_wps_package.py

```diff
@@ -48,8 +48,15 @@
 IANA_ZIP = IANA_NAMESPACE + ":" + CONTENT_TYPE_APP_ZIP  # noqa # pylint: disable=unused-variable
 
 KNOWN_PROCESS_DESCRIPTION_FIELDS = {
-    "id", "title", "abstract", "inputs", "outputs", "executeEndpoint", "keywords", "metadata", "visibility"
+    "id", "title", "abstract", "keywords", "metadata", "inputs", "outputs", "executeEndpoint", "visibility"
 }
+# intersection of fields in InputType and specific sub-schema LiteralInputType
+KNOWN_PROCESS_DESCRIPTION_INPUT_DATA_FIELDS = {
+    "id", "title", "abstract", "keywords", "metadata", "links", "literalDataDomains", "additionalParameters",
+    "minOccurs", "maxOccurs"
+}
```

Comment:
How about using `set.intersection()`, it might be better to avoid repetition

Reply:
Can you elaborate on this? I'm not sure which repetition you refer to?

Reply:
Some of the values are already defined at line 51 in the `KNOWN_PROCESS_DESCRIPTION_FIELDS ` set, since you are doing an intersection, of fields, as described in line 53, the suggestion is to use the `set.intersection()` method with the set above and a set of `LiteralInputType`.
If you update one of the fields of a set, the intersection will be updated automatically 😄

Reply:
Ok. I see what you mean. The values in the process description are not the ones being intersected with though. 
They just match the same names by chance, but are in completely different location in the body.